src/sas/qtgui/MainWindow/GuiManager.py

```diff
@@ -1148,7 +1148,7 @@ def showFitResults(self, output_data):
         Show bumps convergence plots
         """
         self.results_frame.setVisible(True)
-        if output_data:
+        if output_data and len(output_data) > 0 and len(output_data[0]) > 0:
```

Comment:
Is there any way `output_data` can be a scalar quantity if it has any truthiness?